server/scripts/testOCRFromProcessFile.ts

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bun
+
+import { promises as fs } from "fs"
+import path from "path"
+import { chunkByOCRFromBuffer } from "@/lib/chunkByOCR"
+
+type EnvMap = Record<string, string>
+const DEFAULT_TEST_PDF = ""
+
+async function loadEnvFile(envPath: string): Promise<EnvMap> {
+  try {
+    const raw = await fs.readFile(envPath, "utf8")
+    const map: EnvMap = {}
+    for (const line of raw.split(/\r?\n/)) {
+      const trimmed = line.trim()
+      if (!trimmed || trimmed.startsWith("#")) continue
+      const eq = trimmed.indexOf("=")
+      if (eq === -1) continue
+      const key = trimmed.slice(0, eq).trim()
+      let val = trimmed.slice(eq + 1).trim()
+      // Strip surrounding quotes if present
+      if (
+        (val.startsWith('"') && val.endsWith('"')) ||
+        (val.startsWith("'") && val.endsWith("'"))
+      ) {
+        val = val.slice(1, -1)
+      }
+      map[key] = val
+      if (!(key in process.env)) {
+        process.env[key] = val
+      }
+    }
+    return map
+  } catch (err: any) {
+    if (err?.code !== "ENOENT") {
+      console.warn(`Warning: failed to read env file at ${envPath}:`, err)
+    }
+    return {}
+  }
+}
+
+function resolvePdfPath(args: string[], envs: EnvMap): string {
+  // Priority: CLI arg -> TEST_PDF_PATH -> PDF_PATH
+  const cli = args[0]
+  const fromEnv =
+    envs["TEST_PDF_PATH"] ||
+    envs["PDF_PATH"] ||
+    process.env.TEST_PDF_PATH ||
+    process.env.PDF_PATH
+  const p = cli || fromEnv || DEFAULT_TEST_PDF
+  return path.resolve(p)
+}
+
+async function main() {
+  console.log("=== OCR PDF Chunker (processFile simulation) ===")
+
+  // Load env from server/.env (preferred) then from project .env (optional)
+  const cwd = process.cwd()
+  const serverEnvPath = path.resolve(cwd, "server/.env")
+  const rootEnvPath = path.resolve(cwd, ".env")
+  const envs = {
+    ...(await loadEnvFile(serverEnvPath)),
+    ...(await loadEnvFile(rootEnvPath)),
+  }
+
+  // Resolve PDF path
+  const argv = process.argv.slice(2)
+  const pdfPath = resolvePdfPath(argv, envs)
```

Comment:
there was no reason to to complicate pdf path fetch through env 
you could have assumed a folder and directly used it in test script